.github/workflows/ci-cd.yml

```
@@ -85,11 +85,35 @@ jobs:
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.19.0
+
+      - name: Install and migrate
+        env:
+          DATABASE_URL: ${{ secrets.RDS_DATABASE_URL }}
+        run: |
+          pnpm install
+          npx drizzle-kit push
+
+  deploy-production:
```

actionlint (1.7.4)

job "deploy-production" needs job "run-migrations" which does not exist in this workflow

(job-needs)